app/models/rule.rb

```diff
@@ -41,6 +41,10 @@ class Rule < ApplicationRecord
     joins(:profile_rules).where.not(profile_rules: { profile_id: nil }).distinct
   }
 
+  scope :canonical, lambda {
+    where(benchmark_id: ::Xccdf::Benchmark.latest.pluck(:id))
```

Comment:
I didn't limit this to RHEL7 only because I found during testing that some number rules with remediations are part of the RHEL 8 benchmark only, e.g `xccdf_org-ssgproject-content_rule_package_audispd `

Reply:
This makes sense, but I don't like the 'canonical' name. So far, that has meant anything that we've created rather than users. This also returns duplicate rules, which isn't super useful for our purposes. What if replaced this with a `latest` scope, or similarly named, that gives a unique list of rules from the latest benchmarks?

Reply:
This still returns duplicate rules, but I think it's fine as is.